### ISSUE TITLE
Theme does not save across sessions

### ISSUE THREAD

**Issue by willresen** [reduxjs/redux-devtools#1036]
The trace middleware appears to default to the dark theme, but the devtools default to the light theme:

![image](https://user-images.githubusercontent.com/49418430/150200677-65171f08-e893-4af7-8856-ef40bccffc7a.png)

Each session resets the theme preference, so I need to manually set the theme color every time. Am I missing something?

**Comment by Methuselah96** [comment]
Just to clarify, is the bug that you're reporting that the colors for the trace tab aren't right for the light theme?

I'm not sure what you mean by "each sessions resets the theme preference." It seems like the theme preferences are persisted between sessions for me.

**Comment by willresen** [comment]
@Methuselah96, that is correct, the trace colors for the light theme are not right, they are always dark. As for the session resetting, perhaps I am doing something wrong. When I switch from light to dark theme, close Chrome, and then reopen Chrome, it has reset back to the light theme. This means I then have to manually switch back to the dark theme whenever I need the trace tab because it is otherwise unreadable due to the black background.

**Comment by jeevan900929** [comment]
+1

I am only able to see the text when highlighted
![React Redux App - Google Chrome 2022-05-19 19 59 4](https://user-images.githubusercontent.com/2005277/169288577-c03350fa-ccdd-4d45-b22d-ce55e665e708.png)

**Comment by drpicox** [comment]
+1

**Comment by sbimochan** [comment]
My eyes are itching! Is there a fix?

**Comment by nhristova** [comment]
The only way I have found is to switch the Redux DevTools to dark theme... then the text is visible.

![image](https://github.com/reduxjs/redux-devtools/assets/19491111/ddc68413-07f3-4577-a764-bcb576932893)

**Comment by IliyaBrook** [comment]
https://github.com/IliyaBrook/redux-devtools

I fixed this in my version, as it's impossible to work on this black background. I didn't send a PR since not everyone faces this issue, and my correction maintains the same style as in the screenshot, regardless of the theme. Also, in my thread, there's functionality for copying objects and other edits for which there's a PR, but it hasn't been confirmed yet.
![chrome_yQ7fPUpCY3](https://github.com/reduxjs/redux-devtools/assets/60712617/54cd306c-347b-4a69-b25c-c9c908be101f)
